src/components/ChatEntry.jsx

```diff
@@ -1,20 +1,39 @@
 import './ChatEntry.css';
+import PropTypes from 'prop-types';
+import TimeStamp from './TimeStamp';
+
+const ChatEntry = ({ id, sender, body, timeStamp, liked, onLiked }) => {
+  const onHeartClick = () => {
+    onLiked(id);
+  };
+
+  const filledHeart = liked ? '❤️' : '🤍';
+
+  const setMessageBubble = id%2 ? 'local' : 'remote';
 
-const ChatEntry = () => {
   return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
-      <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
-      </section>
-    </div>
+    <>
+      <div className={`chat-entry ${setMessageBubble}`}>
+        <h2 className="entry-name">{sender}</h2>
+        <section className="entry-bubble">
+          <p>{body}</p>
+          <p className="entry-time">
+            <TimeStamp time={timeStamp}></TimeStamp>
+          </p>
+          <button className="like" onClick={onHeartClick}>{filledHeart}</button>
+        </section>
+      </div>
+    </>
   );
 };
 
 ChatEntry.propTypes = {
-  // Fill with correct proptypes
+  id: PropTypes.number.isRequired,
+  sender: PropTypes.string.isRequired,
+  body: PropTypes.string.isRequired,
+  timeStamp: PropTypes.string.isRequired,
+  liked: PropTypes.bool.isRequired,
+  onLiked: PropTypes.func.isRequired,
```

Comment:
Nice use of `Prop-Types` and flagging those necessary for render with `isRequired`!.